ocf_data_sampler/select/dropout.py

```diff
@@ -42,13 +42,16 @@ def draw_dropout_time(
 
 def apply_dropout_time(
     ds: xr.DataArray,
-    dropout_time: pd.Timestamp,
+    dropout_time: pd.Timestamp | None,
 ) -> xr.DataArray:
     """Apply dropout time to the data.
 
     Args:
         ds: Xarray DataArray with 'time_utc' coordinate
         dropout_time: Time after which data is set to NaN
     """
-    # This replaces the times after the dropout with NaNs
-    return ds.where(ds.time_utc <= dropout_time)
+    if dropout_time is None:
+        return ds
+    else:
+        # This replaces the times after the dropout with NaNs
+        return ds.where(ds.time_utc <= dropout_time)
```

Comment:
Small syntax comment, the else isn't strictly needed, could be: 
```suggestion
    if dropout_time is None:
        return ds
    # This replaces the times after the dropout with NaNs
    return ds.where(ds.time_utc <= dropout_time)
```

Reply:
Agreed the else isn't necessary but i think it could be clear with the else for new contributors so leaving it in for now.